pkg/server/server.go

```diff
@@ -553,100 +585,66 @@ func (s *Server) Run(ctx context.Context) error {
 		}
 	}
 
-	if s.options.HomeWorkspaces.Enabled {
+	if s.Options.HomeWorkspaces.Enabled {
```

Comment:
not your change: but why does enableAll not trigger this one?

Reply:
it's not a controller, but a feature. I would actually propose to remove `EnableAll` and only allow opt-out (in follow-up).